templates/components/ui/html/chat/hooks/use-config.ts

```diff
@@ -12,7 +12,15 @@ export function useClientConfig(): ChatConfig {
   const [config, setConfig] = useState<ChatConfig>();
 
   const backendOrigin = useMemo(() => {
-    return chatAPI ? new URL(chatAPI).origin : "";
+    if (chatAPI) {
+      return chatAPI || "";
```

Comment:
seems that is needed
```suggestion
      return new URL(chatAPI).origin;
```